integration/spark/src/main/scala/org/apache/carbondata/view/MVEvents.scala

```diff
@@ -20,31 +20,43 @@ package org.apache.carbondata.view
 import org.apache.spark.sql.SparkSession
 import org.apache.spark.sql.catalyst.TableIdentifier
 
-import org.apache.carbondata.events.Event
+import org.apache.carbondata.events.{Event, MVEventsInfo}
 
 /**
- * For handling operation's before start of mv creation
+ * For handling operation's before start of MV creation
  */
-case class CreateMVPreExecutionEvent(sparkSession: SparkSession,
-    tableIdentifier: TableIdentifier) extends Event
+case class CreateMVPreExecutionEvent(
+    sparkSession: SparkSession,
+    storePath: String,
```

Comment:
Can rename `storePath` to `systemDirectoryPath` in all mv events

Reply:
done